README.md

```diff
@@ -4,6 +4,13 @@
 
 A cd command that learns your habits
 
+## Table of contents
```

Comment:
This is a great addition to the README. Could you create a separate PR for this?

Reply:
Sure!